Workstream Policy.md

```diff
@@ -276,7 +276,7 @@ The [Steering Group]'s determination in each instance is final and non-appealabl
 ### Notice for Living Standards
 
 [Living Standards][Living Standard] must include the following notice, including the link:
-> Copyright © YEAR WHATWG (Apple, Google, Mozilla, Microsoft). This work is licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/).
+> Copyright © YEAR WHATWG (Apple, Google, Mozilla, Microsoft). This work is licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/). Alternatively, this work is licensed under a [BSD 3-Clause License] for incorporation into software.
```

Comment:
Should `https://creativecommons.org/licenses/by/4.0/` also be a link to whatwg.org?

Reply:
We could, but https://creativecommons.org/licenses/by/4.0/ is more readable than what we could achieve. I think the reason we're not linking https://opensource.org/licenses/BSD-3-Clause is because it includes the `Copyright <YEAR> <COPYRIGHT HOLDER>` template-y language.

Reply:
I see, I guess that's similar to https://www.w3.org/Consortium/Legal/2008/03-bsd-license.html. It might be nice to similarly note where it is derived from (though with a more up-to-date link).

Reply:
In our case the wording is identical and not really "derived", while the W3C version changes "copyright holder" to "W3C" in one place. Happy to link to https://opensource.org/licenses/BSD-3-Clause though, but with what wording?

Reply:
At the end of BSD Markdown file I was thinking:

`NOTE: Text from [OSI's 3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause).`

Reply:
Made this more concrete in https://github.com/whatwg/sg/pull/115#discussion_r455842960.